Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListAsGridControlObjectTest.cs

```diff
@@ -661,6 +661,28 @@ public void GetValidationErrors_CellValidationFailure ()
       Assert.That(bocListValidationFailures, Is.EqualTo(expectedValidationFailures));
     }
 
+    [Test]
+    public void GetValidationErrors_CellValidationFailure ()
+    {
+      var home = Start();
+      var bocList = home.ListAsGrids().GetByLocalID("JobList_Validation");
+
+      home.WebButtons().GetByLocalID("ValidationTestCaseCellButton").Click();
+
+      var expectedValidationFailures = new []
+                                       {
+                                           new BocListValidationError(
+                                               "Localized cell validation failure message",
+                                               "89dc8cd2-30e0-4bb3-92a0-4587f32492f5",
+                                               "DisplayName",
+                                               "89dc8cd2-30e0-4bb3-92a0-4587f32492f5",
+                                               "DisplayName")
+                                       };
+
+      var bocListValidationFailures = bocList.GetRow(1).GetCell().WithColumnTitle("DisplayName").GetValidationErrors();
+      Assert.That(bocListValidationFailures, Is.EqualTo(expectedValidationFailures));
```

Comment:
I do prefer this style to inling the variable into the Assert call due to the length / compexity of the actual call. Do you want to update the other usages in this changeset (BocListAsGridControlObjectTest.cs BocListControlObjectTest.cs